pe/management/commands/run.py

```diff
@@ -0,0 +1,40 @@
+# standard libraries
+import logging, os, sys
+from logging import Logger
+
+# third-party libraries
+from django.core.management.base import BaseCommand
+from umich_api.api_utils import ApiUtil
+
+# local libraries
+from pe.main import main
+from constants import API_CONFIG_PATH
+
+
+LOGGER: Logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django management command used for launching the process defined in the main module.
+    """
+
+    def handle(self, *args, **options) -> None:
+        """
+        Entrypoint method required by BaseCommand class (see Django docs).
+        Checks whether the ApiUtil instance is properly configured, invoking the main function if so
+        and exiting if not.
+        """
+        try:
+            api_util: ApiUtil = ApiUtil(
+                os.getenv('API_DIR_URL', ''),
+                os.getenv('API_DIR_CLIENT_ID', ''),
+                os.getenv('API_DIR_SECRET', ''),
+                API_CONFIG_PATH
+            )
+        except Exception as e:
+            LOGGER.error(e)
+            LOGGER.error('api_util was improperly configured; the program will exit.')
+            sys.exit(1)
+
+        main(api_util)
```

Comment:
Nice use of BaseCommand of Django for wiring this!

Reply:
Why can't the `main` when called be instance of class instead of Function/method?

Reply:
It could be a class, but I guess I didn't see a reason why it specifically needed to be one. The `main` code doesn't need to share any data (as instance variables) between multiple methods (there's just one process invoked). Do you see any reason why a class would be a definite improvement over a function?

Reply:
I'm going to stick with the current implementation for now. My thinking is kinda along the lines of this, which @jonespm likes to cite: https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_useless_classes.html?highlight=class